R/pipe_helpers.R

```diff
@@ -89,3 +89,5 @@ pipeline_on_exit <- function(env) {
 
   do.call(on.exit, list(as.call(list(exit_clo)), add = TRUE), envir = env)
 }
+
+utils::globalVariables(".jq_exitfun")
```

Comment:
thanks!

Reply:
No worries, sorry again for the late notice!

Reply:
No worries. Thanks for the PR. 

One more question: Just noticed that on R-devel locally `2020-11-12 r79414` I get a note with 

```
Possibly missing '()' after 'return' in the following function:
  ‘pipeline_on_exit’
```

I assume the note refers to this line https://github.com/ropensci/jqr/blob/master/R/pipe_helpers.R#L86 

Any ideas? Maybe you've encountered this note before. I've not seen this previously anywhere

Reply:
never mind, got it sorted

Reply:
I've never seen this. I assume it's a note from the compiler? And I see you could fix it by quoting with `""`, phew.

Note that this does change the scoping, now if `return` has been redefined in the environment, it will resolve to the wrong function.

Reply:
> now if `return` has been redefined in the environment

I imagine that won't happen very often